WikiExtractor.py

```diff
@@ -3187,6 +3252,20 @@ def main():
     options.expand_templates = args.no_templates
     options.filter_disambig_pages = args.filter_disambig_pages
     options.keep_tables = args.keep_tables
+    options.no_doc = args.no_doc
+    options.no_title = args.no_title
+    options.point_separated = args.point_separated
+    options.blank_line_between_docs = args.blank_line_between_docs
```

Comment:
Already in place by another name as default. No use case for not separating docs has shown up. It can be done in postprocessing, so coding is not strictly useful at this point.